generators/app/templates/copy/components/02-atoms/button/button.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Button from './button.twig';
```

Comment:
Can we use naming convention similar to this 
![image](https://user-images.githubusercontent.com/2114712/141125062-64aa97a3-0e33-44a4-a9eb-29a93905f0c7.png)